client/my-sites/themes/theme-showcase.jsx

```diff
@@ -177,7 +152,34 @@ class ThemeShowcase extends Component {
 
 	isThemeDiscoveryEnabled = () => config.isEnabled( 'themes/discovery' );
 
-	getDefaultStaticFilter = () => defaultStaticFilter;
+	translatedStaticFilters() {
```

Comment:
Nit: Having "translated" in the method name seems a bit redundant.

Reply:
I needed a name to differentiate it from the `staticFilters` constant that was being defined within the methods.